gradebook/src/GradeBook/Statistics.cs

```diff
@@ -0,0 +1,9 @@
+namespace GradeBook
+{
+    public class Statistics
```

Comment:
I don't think anyone else should be creating instances of `Statistics` - I would add an `internal` constructor to keep them from doing so, or a `private` constructor and a `static` method to create an instance.